WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -736,6 +738,7 @@ private void stopActivityTransitionTimer() {
             }
 
             mIsInBackground = false;
+            sAppIsInTheBackground = false;
```

Comment:
Do we really need two variables here which hold the same information?

Reply:
oops, addressed in ca2e6ef